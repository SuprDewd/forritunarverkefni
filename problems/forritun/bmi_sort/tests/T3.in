5

Jónas
1.7
90

Magnús
1.8
80

Bjarki
1.78
60

Þórir
1.5
50

Gunnar
2.1
112
