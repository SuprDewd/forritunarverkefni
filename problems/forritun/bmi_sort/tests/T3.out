Bjarki 18.9
Þórir 22.2
Magnús 24.7
Gunnar 25.4
Jónas 31.1
